tests/testthat/test-modify-model-field.R

```diff
@@ -354,3 +354,50 @@ test_that("remove_tags() works correctly [BBR-MMF-022]", {
   )
   expect_identical(new_mod[[YAML_TAGS]], ref_tags)
 })
+
+test_that("Checking that the star_add function is adding boolean data to YAML[BBR-MMF-023]", {
+  temp_mod_path <- create_temp_model()
+
+  # make a model object and check YAML
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- add_star(new_mod, TRUE)
+
+  expect_true(new_mod$starred)
+
+
+})
+
+test_that("Checking that the star_remove function is removing boolean data from YAML[BBR-MMF-023]", {
+  temp_mod_path <- create_temp_model()
+
+
+  # make a model object and replace the tags
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- add_star(new_mod, TRUE)
+
+
+  new_mod <- remove_star(new_mod, FALSE) %>% suppressWarnings()
```

Comment:
What's the warning here?

Reply:
`Warning message: In modify_model_field(.mod = .mod, .field = YAML_STAR, .value = .star,  :   `star` does not contain any of the following, so they cannot be removed: FALSE`

Reply:
Thanks.

If you're testing that the expected warning is given, it'd be clearer to use `expect_warning()`.  Though, as I mention elsewhere, I'm confused by the `remove_star(mod, FALSE)` thing and think it should just be `remove_star(mod)`, so this test becomes irrelevant if you go that direction.